### PR TITLE
refactor: error view tests and readme

### PR DESCRIPTION
**Pull request by colincclark**
- add readme
- improve snapshots

### CODE REVIEW

File: packages/error-view/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Error View
+
+The error view package is a React
```
Comment (craigbilner) [inline packages/error-view/README.md]: While it's a simple wrapper around the `ErrorBoundary` it does provide a `renderProp` interface so perhaps a "How to use" would be useful?

Comment (colincclark) [reply packages/error-view/README.md]: done

File: packages/error-view/__tests__/shared.js [hunk 2]
```diff
@@ -18,49 +19,54 @@ module.exports = () => {
...
+          <ErrorView>
+            {({ hasError, error }) =>
+              hasError ? <ErrorState error={error} /> : <ThrowsError />
+            }
+          </ErrorView>
+        );
 
-    expect(tree).toMatchSnapshot();
-  });
+        expect(testInstance.toJSON()).toMatchSnapshot();
```
Comment (craigbilner) [inline packages/error-view/__tests__/shared.js]: FYI - `toJSON` is not required because the way it's coerced to a string is the same

Comment (colincclark) [reply packages/error-view/__tests__/shared.js]: removed

**Pull request merged**